src/sentry/integrations/jira/integration.py

```diff
@@ -392,13 +392,21 @@ def get_create_issue_config(self, group, **kwargs):
         kwargs['link_referrer'] = 'jira_integration'
         fields = super(JiraIntegration, self).get_create_issue_config(group, **kwargs)
         params = kwargs.get('params', {})
+
```

Comment:
This is probably an idle complaint for another time, though I wish some of these functions were more modular. Parsing them is a bit daunting.

Reply:
Yeah some of them have multiple things going on. We can fix it gradually though :smile: